posthog/api/__init__.py

```diff
@@ -57,6 +58,7 @@ def api_not_found(request):
 )
 projects_router.register(r"annotations", annotation.AnnotationsViewSet, "project_annotations", ["team_id"])
 projects_router.register(r"feature_flags", feature_flag.FeatureFlagViewSet, "project_feature_flags", ["team_id"])
+projects_router.register(r"prompts", prompt.PromptSequenceStateViewSet, "project_feature_flags", ["team_id"])
```

Comment:
`basename` needs to be updated
```suggestion
projects_router.register(r"prompts", prompt.PromptSequenceStateViewSet, "project_prompts", ["team_id"])
```

As for the shape of the API though, wouldn't it be more intuitive for prompts to be scoped per user instead per project? (mentioned this in a comment above too)